src/admin.controller.ts

```diff
@@ -71,7 +71,7 @@ export class DefaultAdminController {
 
   async getAdminModels(query: AdminModelsQuery): Promise<AdminModelsResult> {
     // @ts-ignore
-    const result: AdminModelsResult = {}
+    const result: Partial<AdminModelsResult> = {}
```

Comment:
Why? We should retur a `AdminModelsResult`, it shouldn't be partial

Reply:
While we're creating it though, it's partial (typescript is sad)